src/components/board/board.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import "../../styles/styles.css";
+import Square from "../square/square";
+
+
+const files = ["a", "b", "c", "d", "e", "f", "g", "h"];
+const ranks = [1, 2, 3, 4, 5, 6, 7, 8];
+
+function makeBoard(ranks, files) {
+  return ranks.reduce((board, rank) => {
+    files.forEach(file => {
+      const id = `${file}${rank}`;
+      board.push({ id: id, piece: null, color: "white" });
+    });
+    return board;
+  }, []);
+}
+
+const board = makeBoard(ranks, files);
+console.log(board);
+
+export default function Board() {
```

Comment:
@RatioChris Is it better to maybe rename makeBoard to prepBoard and move the contents of Board to a function like makeBoard, that just returns the content? I just wonder if that will help it be more easy to test. It's probably easy to test the UI aspect using something other than unit tests. 

```
export default function Board () {
  return makeBoard()
}
```